lib/gears/surface.lua

```diff
@@ -67,60 +67,60 @@ end
 --- Try to convert the argument into an lgi cairo surface.
 -- This is usually needed for loading images by file name and uses a cache.
 -- In contrast to `load()`, errors are returned to the caller.
--- @param _surface The surface to load or nil
+-- @param surface The surface to load or nil
```

Comment:
Looks like it's not called `surface` but `self`.

(I thought ldoc would complain for this)